daemon/bridge.go

```diff
@@ -196,6 +196,7 @@ func ConnectionRPCHandler(d *Daemon) {
 		case ConnectionDelete:
 			DeleteConnection(c.Connection.ConnectionDetails)
 			delete(d.Connections, c.Connection.ContainerID)
+			c.Result <- c.Connection
```

Comment:
Its a good idea to push the result back to the application. But by introducing this, we have to make sure that the non-powerstrip mode code-flow also works fine. If we dont listen to this channel in the non-powerstrip mode, it will result in goroutine deadlock I think. Can you please check that ?